SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -70,12 +68,10 @@ class SPATIALGDK_API InterestFactory
 
 	// Defined Constraint AND Level Constraint
 	void AddPlayerControllerActorInterest(Interest& OutInterest, const AActor* InActor, const FClassInfo& InInfo) const;
-	// Self interests require the entity ID to know which entity is "self". This would no longer be required if there was a first class self
-	// constraint. The components clients need to see on entities they are have authority over that they don't already see through
-	// authority.
-	void AddClientSelfInterest(Interest& OutInterest, const Worker_EntityId& EntityId) const;
+	// The components clients need to see on entities they are have authority over that they don't already see through authority.
```

Comment:
I realise this was me but
```suggestion
	// The components clients need to see on entities they have authority over that they don't already see through authority.
```